crates/bevy_pbr/src/render/parallax_mapping.wgsl

```diff
@@ -5,8 +5,7 @@
     mesh_bindings::mesh
 }
 
-fn sample_depth_map(uv: vec2<f32>, instance_index: u32) -> f32 {
-    let slot = mesh[instance_index].material_bind_group_slot;
+fn sample_depth_map(uv: vec2<f32>, material_bind_group_slot: u32) -> f32 {
```

Comment:
Why did this change? Was it missed in an earlier PR?

Reply:
Not quite sure what you mean. It changed because we don't use that `mesh` array in visbuffer resolution. If you're asking whether I didn't fix meshlets in an earlier PR, yes, I broke them (that's the whole point of this PR).

Reply:
Ah nvm I see that it used to get the slot in the function itself, and all you did was move it outside the function so that the slot could be obtained differently for meshlet meshes.

Reply:
Right.